app/src/main/res/values-de/strings.xml

```diff
@@ -333,53 +333,53 @@
-    <string name="node_port_numeric">Must be numeric</string>
-    <string name="node_port_range">Must be 1&#8211;65535</string>
-    <string name="node_fab_add">Add Node</string>
-    <string name="node_refresh_hint">Touch to refresh!</string>
-    <string name="node_test_error">CONNECTION ERROR %1$d</string>
-    <string name="node_general_error">CONNECTION ERROR</string>
-    <string name="node_auth_error">AUTHENTICATION FAILED</string>
-    <string name="node_result_label">Test Result:</string>
-    <string name="node_result">Height: %1$s (v%2$d), Ping: %3$.0fms, IP: %4$s</string>
-    <string name="node_testing">Testing IP: %1$s &#8230;</string>
-    <string name="node_refresh_wait">Please wait for scan to finish</string>
-    <string name="node_create_hint">Touch to select or add nodes</string>
-    <string name="node_pull_hint">Add nodes manually or pull down to scan</string>
-    <string name="node_scanning">Scanning network&#8230;</string>
-    <string name="node_nobookmark">Automatically bookmarked best %1$d nodes</string>
-    <string name="label_test">Test</string><!--note: as in "Test a network connection"-->
-
-    <string name="send_address_resolve_bip70">Resolving Payment URI&#8230;</string>
-    <string name="send_address_not_bip70">Could not resolve Payment URI</string>
-    <string name="send_address_bip70">Resolved Payment URI &#x2714;</string>
+    <string name="node_user_hint">Nutzername (Optional)</string>
+    <string name="node_pass_hint">Passwort (Optional)</string>
+    <string name="node_host_unresolved">Kann Webadresse nicht auflösen</string>
+    <string name="node_host_empty">Wir brauchen das!</string>
+    <string name="node_port_numeric">Muss eine Zahl sein</string>
+    <string name="node_port_range">Muss 1&#8211;65535 sein</string>
+    <string name="node_fab_add">Node hinzufügen</string>
+    <string name="node_refresh_hint">Tippe zum aktualisieren!</string>
+    <string name="node_test_error">VERBINDUNGSFEHLER %1$d</string>
+    <string name="node_general_error">VERBINDUNGSFEHLER</string>
+    <string name="node_auth_error">AUTHENTIFIZIERUNG FEHLGESCHLAGEN</string>
+    <string name="node_result_label">Testergebnis:</string>
+    <string name="node_result">Höhe: %1$s (v%2$d), Ping: %3$.0fms, IP: %4$s</string>
+    <string name="node_testing">Teste IP: %1$s &#8230;</string>
+    <string name="node_refresh_wait">Warte bitte, bis die Suche abgeschlossen ist.</string>
+    <string name="node_create_hint">Tippe, um Nodes auszuwählen oder hinzuzufügen</string>
+    <string name="node_pull_hint">Füge manuell Nodes hinzu oder ziehe nach unten, um zu suchen</string>
+    <string name="node_scanning">Durchsuche Netzwerk&#8230;</string>
+    <string name="node_nobookmark">Beste %1$d Nodes automatisch vorgemerkt</string>
+    <string name="label_test">Testen</string><!--note: as in "Test a network connection"-->
+
+    <string name="send_address_resolve_bip70">Löse Zahlungs-URI&#8230; auf</string>
```

Comment:
`&#8230;` is unicode for `...` so it should be: `Löse Zahlungs-URI auf&#8230;`

Reply:
Oh.. good spot